docs/installation.md

```diff
@@ -95,7 +102,10 @@ To install using Cocoapods, simply insert the following line into your `Podfile`
        setContentView(mReactRootView);
    }
    ```
-4. specify your Google Maps API Key in your `AndroidManifest.xml`:
+4. specify your Google Maps API Key:
+    > To develop is recommended a ***Browser Key*** without refeer restriction. Go to https://console.developers.google.com/apis/credentials to check your credentials.
```

Comment:
Yes. For develop mode it's easy to use browser key because it's not necessary to add any refeer  restriction. For Android API key you need to add the sign of your app.